.gitlab/quartz-baseline.yml

```diff
@@ -92,8 +92,10 @@ report_baseline:
         if [[ "$AUTOTEST_COMMIT" != "NO" ]]; then
           git pull && \
           git add ${rundir} && \
-          git commit -m "${msg}" && \
-          git push origin master
+          git commit -m "${msg}"
+          if [[ $? == 0 ]]; then
+            ${CI_PROJECT_DIR}/.gitlab/scripts/git_try_to_push
+          fi
```

Comment:
Maybe simplify this a bit:
```suggestion
          git commit -m "${msg}" && \
          ${CI_PROJECT_DIR}/.gitlab/scripts/git_try_to_push
```

Reply:
Good point, that was an artifact from before factoring out `git_try_to_push` to its own script